index-alphabetic-browse.sh

```diff
@@ -31,11 +31,17 @@ then
   SOLR_HOME="$VUFIND_HOME/solr/vufind"
 fi
 
+# This would be /opt/bitnami/solr on a solr docker container
+if [ -z "$BITNAMI_SOLR_HOME" ]
```

Comment:
Agreed that it seems confusing to key this to Bitnami explicitly. And for non-Bitnami sites, the `CLASSPATH` below could have unexpected side effects.

Reply:
1.) Why is solrmarc_core.jar included in this PR? Did you mean to replace the browse handler jars instead?
It needs to be in the classpath for both indexing and browsing. I did not change the browse handler jar because I assumed you do that before every release (I can include it if you wish).
2) I see, yes, we would need a local implementation for non-MARC input. This goes beyond what I intended to do in the scope of resolving this issue. I could possibly work on it later, but not right away.

Reply:
I think `solrmarc_core.jar` needs to be in `solr/vufind/jars` in the release. How could I copy the file from `import` ? Would `build.xml` be a good place for that ?

Reply:
You should be able to add it to the classes included in solrconfig.xml without making an extra copy. Not at a computer right now, but I can dig up the details tomorrow if I'm being too vague right now. :-)

Reply:
I see, there is already that in `solrconfig.xml`:
`<lib dir="../../../import/lib" regex="marc4j.*\.jar" />`
So I could add .../import for solrmarc. The dependency of the solr config to vufind is not ideal for us because we run them separately, but this change won't affect that dependency much. I will also have to add `solrmarc_core.jar` to the classpath in `index-alphabetic-browse.sh`.